src/components/module_parser.js

```diff
@@ -51,7 +52,7 @@ const ModuleParser = {
         const isStartInAutumn = ModuleParser.isAutumnSemester(firstModule.anlassnumber);
 
         const lastPart = hsluModuleName.split('.')[2];
-        const moduleYear = Number('20' + lastPart.substring(1, 3));
+        const moduleYear = Number('20' + (lastPart != undefined ? lastPart.substring(1, 3) : '00'));
```

Comment:
A few lines below this we have a check to see if `isModuleInAutumn` is `undefined`. This will be the case for the "I.WS3_GAME_DS" module. In this case we return `undefined` so we don't need to calculate a year.

```js
const moduleYear = lastPart === undefined ? undefined : Number('20' + lastPart.substring(1, 3));
```